dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021,2023-2024 NXP
+ * Copyright 2021,2023-2025 NXP
```

Comment:
Please avoid formulation like `update..`. Please explain exactly what is done and WHY is done.